README.md

```diff
@@ -1,8 +1,64 @@
-# Test Repo
+# EnChord
 
-The chopsticks are the best female developer duo.
+**EnChord** is a perfect tool to develop your music dictation skills. Challenge yourself with different levels of quizzes and follow your progress by checking your highest score and average test results. You can also review what are the frequently mistaken interval types.
 
-## Scripts
+## Table of contents
+
+1. [How it works](https://github.com/TheChopsticks/EnChord/edit/main/README.md#how-it-works)
```

Comment:
```suggestion
1. [How it works](#how-it-works)
```
Since we're linking within the same document, just using the anchor should be sufficient

Reply:
Thank you! I will change them now. Apparently, I love taking a rough road 🤣